covid19-etl/etl/cityofchicago.py

```diff
@@ -113,8 +113,8 @@ def add_to_summary_group_demographics(
 
         # age group mapping for value from original dataset to value in Gen3 data dictionary
         age_group = {
-            "0_17": "less than 17",
-            "18_29": "18 to 29",
+            "0_17": "less than 20",
+            "18_29": "20 to 29",
```

Comment:
that's not great though, ages from 18 to 20 are mapped to the wrong value. i think the dictionary should be updated